test/ClientTest.php

```diff
@@ -7,8 +7,9 @@
 use Http\Discovery\MessageFactoryDiscovery;
 use Http\Mock\Client as MockClient;
 use Ricbra\Knmi\Client;
+use PHPUnit\Framework\TestCase;
 
-class ClientTest extends \PHPUnit_Framework_TestCase
+class ClientTest extends TestCase
```

Comment:
Could you revert this change as it is not required? We're depending on phpunit ^5.6 which still has the old non-namespaced class names. If you run the tests with the composer installed phpunit, it should work with the non-namespaced classes (`vendor/bin/phpunit`).

Reply:
✔️